client/src/questions-n-answers/qNa.jsx

```diff
@@ -257,6 +263,25 @@ class QuestionsNAnswers extends React.Component {
     }
   }
 
+  updateAnswers(answers) {
```

Comment:
I'm guessing you have a plan for using these later-- but figured I'd comment to note just in case they were supposed to be filled out but for some reason were not on the PR